openpype/plugins/publish/collect_anatomy_frame_range.py

```diff
@@ -0,0 +1,36 @@
+import pyblish.api
+
+
+class CollectAnatomyFrameRange(pyblish.api.InstancePlugin):
+    """Collect Frame Range specific Anatomy data.
+
+    Plugin is running for all instances on context even not active instances.
+    """
+
+    order = pyblish.api.CollectorOrder + 0.491
+    label = "Collect Anatomy Frame Range"
+    families = ["plate", "pointcache",
+                "vdbcache", "online",
+                "render"]
+    hosts = ["traypublisher"]
+
+    def process(self, instance):
+        self.log.info("Collecting Anatomy frame range.")
+        asset_doc = instance.data.get("assetEntity")
+        if not asset_doc:
+            self.log.info("Missing required data..")
+            return
+
+        asset_data = asset_doc["data"]
+        for key in (
+            "fps",
+            "frameStart",
+            "frameEnd",
+            "handleStart",
+            "handleEnd"
+        ):
+            if key not in instance.data and key in asset_data:
+                value = asset_data[key]
+                instance.data[key] = value
+
+        self.log.info("Anatomy frame range collection finished.")
```

Comment:
Debug logs
```suggestion
        self.log.debug("Anatomy frame range collection finished.")
```

Reply:
Instead it'd would have been much clearer if instead we would track what keys were set in e.g. a `keys_set` variable and then logged what data was set:
```python
if keys_set:
    self.log.debug(
        "Set instance frame range data from asset"
        "entity for keys: {}".format(", ".join(keys_set)
    )
```
So that the logs actually provide debugging information of what really occurred.